rebalancer.py

```diff
@@ -189,11 +215,13 @@ def auto_enable():
                 channel.auto_rebalance = True
                 channel.save()
                 Autopilot(chan_id=channel.chan_id, peer_alias=channel.alias, setting='Enabled', old_value=0, new_value=1).save()
+                print('Auto Pilot Enabled: ', channel.alias, ' : ', channel.chan_id , ' In: ', iapD, ' Out: ', oapD)
```

Comment:
Here we'd potentially need to save multiple records though or potentially start recording a channel id list here instead?

Reply:
understood

Reply:
https://github.com/cryptosharks131/lndg/pull/94/commits/9cc8480cfe8992af4f6abe753f099ba8142a5d95